Actors/pythonactor.c

```diff
@@ -532,15 +534,20 @@ int python_init()
     //Py_NoUserSiteDirectory = true;
     Py_Initialize();
     //  add the working dir for importing python files
-    rc = PyRun_SimpleString(
+    char *script = (char*)malloc(512 * sizeof(char));
+    // Prints "Hello world!" on hello_world
+    snprintf(script, 512 * sizeof(char),
                     "import sys\n"
-                    "import os\n"
-                    //"print(os.getcwd())\n"
-                    "sys.path.insert(0, \"\")\n"
-                    //"sys.path.append(os.getcwd())\n"
-                    //"sys.path.append('/home/arnaud/src/czmq/bindings/python')\n"
-                    "print(\"Python {0} initialized. Paths: {1}\".format(sys.version, sys.path))\n"
-                       );
+                     "import os\n"
+                     //"print(os.getcwd())\n"
+                     "sys.path.insert(0, \"%s/misc/scripts\")\n"
+                     "sys.path.insert(0, \"\")\n"
+                     //"sys.path.append(os.getcwd())\n"
+                     //"sys.path.append('/home/arnaud/src/czmq/bindings/python')\n"
+                     "print(\"Python {0} initialized. Paths: {1}\".format(sys.version, sys.path))\n",
+                     GZB_GLOBAL.RESOURCESPATH);
```

Comment:
It fails on Windows here because of linker problem. So it compiles fine but it won't link. The weird thing is that it does work in stage.cpp so it might be C compiler thing of MSVC